messages.js

```diff
@@ -1,41 +1,47 @@
     ' • You may enter prices in pp or k (ex: 1100pp or 1.1k).\n' +
     ' • Parser will not detect aliases (ex: watching "Thick Banded Belt" will not detect "TBB"), however this is a future goal.\n' +
     '\n' +
-    '***COMMANDS:***\n' +
-    '!help\n' +
-    '> Displays available commands.\n' +
-    '!add watch: `item`, `maximum price`, `server`\n' +
-    '> Starts a watch based on entered parameters - watches expire after 7 days.  Price is optional.\n' +
-    '!end watch: `item`, `server`\n' +
-    '> Ends a currently running watch.\n' +
-    '!end all watches\n' +
-    '> Ends all currently running watches.\n' +
-    '!extend all watches\n' +
-    '> Extends your current watches another 7 days.\n' +
-    '!show watch: `item`, `server`\n' +
-    '> Lists details for a watch for entered item - if no arguments are provided, behaves as *!show watches*.\n' +
-    '!show watches\n' +
-    '> Lists details for all watches.\n' +
-    '\n' +
-    '***TIPS:***\n' +
-    ' • You can use `!add watch` to update an existing watch if you wish to modify the price and/or reset the 7 day expiration timer.\n' +
-    ' • To report a problem or request a feature, talk to us in #feedback_and_ideas, or create an issue here: https://github.com/jamesjamail'
+    '__***COMMANDS***__\n' +
+    '**!help**\n' +
+    '> Displays available commands.\n\n' +
+    '**!watch `item`, `maximum price`, `server`**\n' +
+    '> Starts a watch based on entered parameters - watches expire after 7 days.  Price is optional.\n\n' +
+    '**!unwatch `item`, `server`**\n' +
+    '> Ends a currently running watch.\n\n' +
+    '**!watches**\n' +
+    '> Returns every watch as an individual message. An optional search term can be specified.  For example:\n `!watches belt of` returns all watches containing \"belt of\".\n\n' +
+    '**!list**\n' +
+    '> Lists details for all watches, account-wide blocks and snoozes in a concise message.\n\n' +
+    '**!block `seller`, `server`**\n' +
+    '>  Blocks a seller from triggering any watch notifications.  `server` is optional- if omitted the seller will be blocked on both servers.\n\n' +
+    '**!unblock `seller`, `server`**\n' +
+    '> Unblocks a seller for all watch notifications.  `server` is optional- if omitted the seller will be unblocked on both servers.\n\n' +
+    '**!snooze `hours`**\n' +
+    '> Pauses notifications on all watches.  \`hours\` is opitional; if omitted, watches are snoozed for 6 hours.\n\n' +
+    '**!unsnooze**\n' +
+    '> Resumes notifications on all watches.\n\n' +
+    '**!extend**\n' +
+    '> Extends your current watches another 7 days.\n\n' +
+    '__***TIPS***__\n' +
+    ' • You can use `!watch` to update an existing watch if you wish to modify the price and/or reset the 7 day expiration timer.\n' +
+    ' • Watch notifications and results from the \`!watches\` command have emoji reactions that trigger useful commands.\n' +
+    ' • To report a problem or request a feature, talk to us in #feedback_and_ideas.'
```

Comment:
Not super important, but trying to remember if this will auto-link the channel or if there's something we need to do to make that a channel link...

Reply:
It does not auto link the channel, I thought it would =/

Reply:
There IS a way to make it link the channel, but it requires a little bit of code. Can worry about it later.